### PR TITLE
[[FIX]][[TEST]] add Comment, DocumentFragment, Range, and Text to browser env

### PR DESCRIPTION
**Pull request by takenspc**
DOM4 defines the constructors for these interfaces.
http://www.w3.org/TR/2014/WD-dom-20140710/

### CODE REVIEW

File: src/vars.js [hunk 1]
```diff
@@ -77,10 +77,12 @@ exports.browser = {
   close                : false,
   closed               : false,
   CustomEvent          : false,
+  Comment              : false,
```
Comment (caitp) [inline src/vars.js]: alphabetically, `o` is before `u` (or `l`)

Comment (caitp) [inline src/vars.js]: then again, it doesn't matter, it's all good =)

**Comment by caitp** [comment]
looks good to me

**Comment by jugglinmike** [comment]
Me too! I'd like to make a few cosmetic changes before landing this:

- Sort the new values alphabetically
- Format the commit message to adhere to [the project's guidelines](https://github.com/jshint/jshint/blob/3c81989d0fda302238608044d6fc191e61f7b9fa/CONTRIBUTING.md#commit-message-guidelines)

I'll do this later today, barring any objection from @takenspc.

**Comment by rwaldron** [comment]
LGTM pending change requests :)

**Comment by takenspc** [comment]
Thank you all. It seems that it was merged as https://github.com/jshint/jshint/commit/32f02e0d4edbb38e1724770b8ea3cce56b4e0888

Should I close this PR?

**Comment by jugglinmike** [comment]
You're right--I landed this just last night. I could have sworn I left a comment and closed this issue, Takeshi. Sorry about that.

Anyway, thanks for the contribution!

**Pull request closed without merging**